recipes/xapian-core/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xapian-core" %}
+{% set version = "1.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://oligarchy.co.uk/xapian/{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 13f08a0b649c7afa804fa0e85678d693fd6069dd394c9b9e7d41973d74a3b5d3
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: |
+    ./configure \
+    CXXFLAGS=-I$PREFIX/include \
+    LDFLAGS=-L$PREFIX/lib \
+    --prefix=$PREFIX
+
+    make && make install
+    rm $PREFIX/share/doc/xapian-core -rf
```

Comment:
When `build/script` exceeds a line or two, it's preferred to use a `build.sh` file.